src/ir_Hitachi.cpp

```diff
@@ -520,3 +521,222 @@ bool IRrecv::decodeHitachiAc424(decode_results *results, const uint16_t nbits,
   return true;
 }
 #endif  // DECODE_HITACHI_AC424
+
+// Class for handling the remote control on a Hitachi_AC424 53 byte A/C message
+IRHitachiAc424::IRHitachiAc424(const uint16_t pin, const bool inverted,
+                         const bool use_modulation)
+    : _irsend(pin, inverted, use_modulation) { stateReset(); }
+
+// Reset to auto fan, cooling, 23° Celcius
+void IRHitachiAc424::stateReset(void) {
+  for (uint8_t i = 0; i < kHitachiAc424StateLength; i++)
+    remote_state[i] = 0x00;
+
+  remote_state[0]  = 0x01;
+  remote_state[1]  = 0x10;
+  remote_state[3]  = 0x40;
+  remote_state[5]  = 0xFF;
+  remote_state[7]  = 0xCC;
+  remote_state[11] = 0x13;  // Button Action
+  remote_state[33] = 0x80;
+  remote_state[35] = 0x03;
+  remote_state[37] = 0x01;
+  remote_state[39] = 0x88;
+  remote_state[45] = 0xFF;
+  remote_state[47] = 0xFF;
+  remote_state[49] = 0xFF;
+  remote_state[51] = 0xFF;
+
+  setTemp(23);
+  setPower(true);
+  setMode(kHitachiAc424Cool);
+  setFan(kHitachiAc424FanAuto);
+  setInvertedStates();
```

Comment:
`setInvertedStates()` can be safely removed. All external interaction with the internal (bytes) state will be corrected via `getRaw()`'s call of it.